docker-compose.yaml

```diff
@@ -0,0 +1,27 @@
+version: "3.3"
+   
+volumes: 
+  pgdata:
+  
+
+services:
+  db:
+    image: postgres
```

Comment:
Specify the image version and the ports, i.e. 5432:5432 and for an instance `postgres:13-alpine`